Spigot-Server-Patches/0010-Adventure.patch

```diff
@@ -610,7 +610,7 @@ index 0000000000000000000000000000000000000000..c60457e9240c33a4721b82a00cef081f
 +    public static ArrayList<Component> asAdventure(final List<IChatBaseComponent> vanillas) {
 +        final ArrayList<Component> adventures = new ArrayList<>(vanillas.size());
 +        for (final IChatBaseComponent vanilla : vanillas) {
-+            adventures.add(asAdventure(vanilla));
++            adventures.add(vanilla == null ? Component.empty() : asAdventure(vanilla));
```

Comment:
This check should probably be inside of `asAdventure(final IChatBaseComponent component` that way it covers a bit of a wider surface

Reply:
I don't know if I understand correctly.

I hope that this is correct.